controllers/newsletter.js

```diff
@@ -0,0 +1,48 @@
+const newsletters = require("../models/newsletter");
+
+const ERROR_MESSAGE = "Something went wrong. Please try again or contact admin";
+
+const addEmail = async (req, res) => {
+  try {
+    const response = await newsletters.subscribe(req.body);
+    return res.json({
+      message: "gotcha",
```

Comment:
Can we please change the message to something like `Email added successfully` ?

Reply:
done